Classes/extra_functions.py

```diff
@@ -303,6 +304,11 @@ def check(reaction, user, valid_approvers):
     # we aren't going to delete unknown messages. Just react with a question mark.
     await message.add_reaction("\N{BLACK QUESTION MARK ORNAMENT}")
 
-def ts_print(*objects, sep=' ', end='\n', file=sys.stdout, flush=False):
-    timestamp = colored(datetime.now().strftime('%b-%d-%Y %H:%M:%S'), 'green')
-    print(timestamp + *objects, sep=sep, end=end, file=file, flush=flush)
+def ts_print(*objects, sep=' ', end='\n', file=stdout, flush=False):
```

Comment:
Instead of having lots of these like sep, end, file... we can use **kwargs to collect all the keyword arguments into a dictionary and then convert them back into keyword arguments with **kwargs when you call print. This will also add support for any other keyword arguments print may have.

Reply:
I try to avoid kwargs when doing functions like this, since it breaks tooltips in most editors.